library/Vanilla/Site/TranslationProvider.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @author Alexander Kim <alexander.k@vanillaforums.com>
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace Vanilla\Site;
+
+use Vanilla\Contracts\Site\TranslationProviderInterface;
+
+/**
+ * Class TranslationProvider
+ * @package Vanilla\Site
+ */
+class TranslationProvider implements TranslationProviderInterface {
+    /**
+     * @inheritdoc
+     */
+    public function supportContentTranslation(): bool {
+        return false;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function translate(string $propertyKey, $sourceValue): string {
+        return Gdn::translate($propertyKey, $sourceValue);
```

Comment:
```suggestion
        return $this->locale->translate($propertyKey, $sourceValue);
```

We should be able to access the same method just by dependency injecting `LocaleInterface`.